main/dconf/APKBUILD

```diff
@@ -1,32 +1,28 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=dconf
-pkgver=0.26.0
+pkgver=0.32.0
 pkgrel=1
 pkgdesc="A low-level configuration system."
 url="http://live.gnome.org/dconf"
 arch="all"
 license="LGPL-2.1"
 depends=""
-makedepends="glib-dev dbus-dev docbook-xsl vala"
+makedepends="glib-dev dbus-dev docbook-xsl vala gtk+3.0-dev libxml2-dev meson py3-setuptools bash-completion"
```

Comment:
```suggestion
makedepends="glib-dev dbus-dev docbook-xsl vala gtk+3.0-dev libxml2-dev meson py3-setuptools"
```

Reply:
No, it checks for bash-completion via pkg-config, presumably to get the right dir to install it to.